modules/ossm-rn-known-issues.adoc

```diff
@@ -34,6 +34,15 @@ These are the known issues in {SMProductName}:
 
 * link:https://github.com/istio/istio/issues/14743[Istio-14743] Due to limitations in the version of Istio that this release of {SMProductName} is based on, there may be applications that are currently incompatible with {SMProductShortName}. See the linked community issue for details.
 
+* https://issues.redhat.com/browse/OSSM-2344[OSSM-2344] Restarting Istiod causes Kiali to flood CRI-O with port-forward requests. This issue occurs when Kiali cannot connect to Istiod and Kiali simultaneously issues a large number of requests to istiod.
```

Comment:
Isn't this one being fixed in 2.3.1? At least the Jira says it's on QA.

Reply:
@jwendell the Jira says unresolved and unassigned, so I thought it wasn't being worked on and would remain a known issue. Reading the comments, yes, looks like Mazz added it to the errata. I will move this to fixed issues instead. Thank you for pointing this out.

Reply:
This has been moved and the language tweaked.